modules/autotagging-commons/common/src/main/java/org/opensearch/rule/autotagging/FeatureType.java

```diff
@@ -49,6 +47,11 @@ public interface FeatureType extends Writeable {
      */
     Map<String, Attribute> getAllowedAttributesRegistry();
 
+    /**
+     * returns the validator for feature value
+     */
+    FeatureValueValidator getFeatureValueValidator();
```

Comment:
Should this return null by default instead of overriding multiple places in the tests?